File: algorithms/2016.maximum-difference-between-increasing-elements/solution.go
func maximumDifference(nums []int) int {
	max := 0

	for i := 0; i < len(nums)-1; i++ {
		for j := i + 1; j < len(nums); j++ {
			if nums[j]-nums[i] > max {
				max = nums[j] - nums[i]
			}
		}
	}

	if max > 0 {
		return max
	} else {
		return -1
	}
}

File: algorithms/2016.maximum-difference-between-increasing-elements/solution.rs
impl Solution {
    pub fn maximum_difference(nums: Vec<i32>) -> i32 {
        let mut max = 0;

        for i in 0..(nums.len() - 1) {
            for j in (i + 1)..nums.len() {
                if nums[j] - nums[i] > max {
                    max = nums[j] - nums[i];
                }
            }
        }

        if max > 0 {
            max
        } else {
            -1
        }
    }
}
